docker-compose.prod.yml

```diff
@@ -0,0 +1,97 @@
+# NOTE: This files shows a possible production setup.
+#       If you are not familiar with handling containers in production, please 
+#       consider a different approach of hosting. This setup can lead to 
+#       errors and loss of production data if handled incorrectly. Currently 
+
+# WARNING: Currently, container based setups are not officially supported by Humhub GmbH.
+#          Use at your own risk!
+version: '3.1'
+services:
+  nginx:
+    build:
+      context: .
+      target: humhub_nginx
+    ports:
+      - "8080:80"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost/ping"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+    volumes:
+      - "uploads:/var/www/localhost/htdocs/uploads"
+      - "assets:/var/www/localhost/htdocs/assets"
+  humhub:
+    build:
+      context: .
+      target: humhub_phponly
+    expose:
+      - '9000'
+    environment:
+      HUMHUB_DB_USER: ${HUMHUB_DB_USER}
+      HUMHUB_DB_PASSWORD: ${HUMHUB_DB_PASSWORD}
+      HUMHUB_DB_HOST: 'db'
+      HUMHUB_DB_NAME: 'humhub'
+      HUMHUB_AUTO_INSTALL: 1
+      # HUMHUB_DEBUG: 1 # you can currently not use debug in prod 
+      HUMHUB_PROTO: 'https'
+      HUMHUB_HOST: 'localhost'
+      # Mailer install setup
+      HUMHUB_MAILER_SYSTEM_EMAIL_ADDRESS: 'noreply@example.com'
+      HUMHUB_MAILER_SYSTEM_EMAIL_NAME: 'HumHub'
+      HUMHUB_MAILER_TRANSPORT_TYPE: 'smtp'
+      HUMHUB_MAILER_HOSTNAME: 'mailer'
+      HUMHUB_MAILER_PORT: 1025
+      #HUMHUB_MAILER_USERNAME:
+      #HUMHUB_MAILER_PASSWORD:
+      #HUMHUB_MAILER_ENCRYPTION:
+      HUMHUB_MAILER_ALLOW_SELF_SIGNED_CERTS: 0
+      # Cache Config
+      HUMHUB_CACHE_EXPIRE_TIME: 3600
+      HUMHUB_CACHE_CLASS: yii\redis\Cache
+      HUMHUB_QUEUE_CLASS: humhub\modules\queue\driver\Redis
+      HUMHUB_REDIS_HOSTNAME: redis
+      HUMHUB_REDIS_PORT: 6379
+      HUMHUB_REDIS_PASSWORD: redis_password
+    volumes:
+      - "uploads:/var/www/localhost/htdocs/uploads"
+      - "assets:/var/www/localhost/htdocs/assets"
+#     - "${HUMHUB_VOLUME_CONFIG}:/var/www/localhost/htdocs/protected/config"
+    healthcheck:
+      test: ["CMD", "php-fpm-healthcheck"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+  db:
+    image: mariadb
```

Comment:
use a tagged version of mariadb with major.minor tag as mariadb:10.4 so it does not pull the latest release